clayui.com/gatsby/createPages.js

```diff
@@ -11,6 +11,9 @@ require('dotenv').config({
 	path: `.env.${process.env.NODE_ENV}`,
 });
 
+const CONTRIBUTING_URL =
+	'https://github.com/liferay/clay/blob/master/CONTRIBUTING.md';
+
```

Comment:
I wouldn't define this because:

- It is used in exactly one place; _and_
- All the other `fromPath`, `toPath` pairs are just written inline as string literals, making this inconsistent.